spec/index.bs

```diff
@@ -5010,37 +5010,41 @@ GPUQueue includes GPUObjectBase;
             <ul dfn-type=argument dfn-for="GPUQueue/writeBuffer(buffer, bufferOffset, data, dataOffset, size)">
                 - |buffer|: {{GPUBuffer}} <dfn>buffer</dfn>
                 - |bufferOffset|: {{GPUSize64}} <dfn>bufferOffset</dfn>
-                - |data|: <code>[{{AllowShared}}] {{ArrayBuffer}}</code> <dfn>data</dfn>
+                - |data|: <code>[{{AllowShared}}] {{BufferSource}}</code> <dfn>data</dfn>
                 - |dataOffset|: optional {{GPUSize64}} <dfn>dataOffset</dfn> = 0
                 - |size|: optional {{GPUSize64}} <dfn>size</dfn>
             </ul>
 
             **Returns:** `void`
 
+            - If |data| is a TypedArray, let the element type be the type of the TypedArray.
+                Otherwise, let the element type be bytes.
```

Comment:
Should "TypedArray" be "ArrayBufferView", or at least reference it parenthetically? https://heycam.github.io/webidl/#BufferSource

Would it be better to be explicit, and say something like "Otherwise, |data| is an ArrayBuffer; let the element type..."?

Reply:
ArrayBufferView includes DataView which does not have an element type, so I used TypedArray even though there doesn't seem to be a linkable spec for `{{TypedArray}}`. Making it more explicit here would clarify it though.

Reply:
Thanks, forgot that DataView was mixed in there. Your clarification sounds good.